pyo3-macros-backend/src/pyclass.rs

```diff
@@ -1038,6 +1037,12 @@ impl<'a> PyClassImplsBuilder<'a> {
                 #dict_offset
 
                 #weaklist_offset
+
+                fn static_type_object() -> &'static _pyo3::impl_::pyclass::StaticTypeObject<Self> {
+                    use _pyo3::impl_::pyclass::*;
```

Comment:
Do we really need the glob import here? Why not just `StaticTypeObject` as in the example code from the guide?

Reply:
Fair point, fixed. 👍